### ISSUE TITLE
Update website docs for gnome-shell-extensions instead of gnome tweaks

### ISSUE THREAD

**Issue by wolftune** [rcaelers/workrave#441]
As I mentioned at https://github.com/rcaelers/workrave/issues/333#issuecomment-1319077180 the website docs need to be updated. If the website itself were in git where I knew how to reach it, I'd submit a PR

**Comment by rcaelers** [comment]
Indeed. The website is in Git: https://github.com/rcaelers/workrave-website. All documentation is in Markdown stored in Git.
I will transfer this issue.
